### ISSUE TITLE
Possible to listen on a different IP than 127.0.0.1 / localhost?

### ISSUE THREAD

**Issue by kaia9** [gwomacks/php-debug#214]
I've got a server running within vagrant on my local machine, which is serving on a local fixed IP address (192.168.x.x) - is it possible through a different config param to setup php-debug to listen to that IP instead of 127.0.0.1 / localhost? something like `ServerIP`?

```
"php-debug":
    ServerPort: 9000
    PathMaps: [
      "remotepath;localpath"
      "/vagrant/src/server;/Users/josh/dev/vagrant/src/server"
    ]
```

**Comment by phearlez** [comment]
I believe you need to do this on the server in Vagrant. In your php.ini file you should have this line:

xdebug.remote_host=127.0.0.1

However if you're going to make that a different IP I presume it's going to need to be one that the server can route to. So unless the vagrant host is also getting an IP in that 192.168.X.X range this may not be doable.

**Comment by developer-lindner** [comment]
I'm also interested in changing the default values. I've got a Docker Setup (on Mac OS X) and need to listen on IP 10.254.254.254 with a different IDE_KEY. Any suggestions ?

**Comment by cchamplin** [comment]
This should be available in 0.2.5
